imesec-full-stack.yml

```diff
@@ -0,0 +1,91 @@
+version: "3.4"
+services:
+  lh-web:
+    image: razgrizone/lightning-hacks-website
+    environment:
+      VIRTUAL_HOST: lh.torradeira.imesec.ime.usp.br

```

Comment:
__Discussion__: Do you think it is better to already leave the VIRTUAL_HOSTs without `.torradeira` so that they're ready to switch whenever, even though they're being shadowed by `batata`'s proxy?